app/src/main/java/comcmput301f16t01/github/carrier/Requests/ElasticRequestController.java

```diff
@@ -200,12 +202,30 @@ protected RequestList doInBackground(Location... search_parameters) {
         }
     }
 
+    // TODO http://stackoverflow.com/questions/1485708/how-do-i-do-a-http-get-in-java
+
+    /**
+     * Searches for a list of possible geo-location from an address string.
+     */
+    public static class SearchByAddressTask extends AsyncTask<String, Void, List<Address>> {
+
+        @Override
+        protected List<Address> doInBackground(String... addresses) {
+            List<Address> addressList = null;
+
+            GeocoderNominatim geoNom = new GeocoderNominatim("");
+            try {
+                addressList = geoNom.getFromLocationName(addresses[0], 50);
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+            return addressList;
+        }
+    }
+
     /**
-     * Get all of a rider's requests including a filter by status
-=======
      * Get all of a rider's requests filtered by status
      * @see RequestController#fetchRequestsWhereRider(User, Integer...)
->>>>>>> 92fda80c294f949b664f49d206e3dedeea1f17a7
```

Comment:
I noticed you caught this here but I commented in your other pull request about it.